src-cpp/rdkafkacpp.h

```diff
@@ -1451,6 +1713,9 @@ class RD_EXPORT Message {
    * @returns \c rd_kafka_message_t*
    */
   virtual struct rd_kafka_message_s *c_ptr () = 0;
+
+  /** @returns The Headers instance for this Message (if applicable) */
```

Comment:
Beef this up. What does "if applicable" mean? What is returned if they are not applicable?